### PR TITLE
Add support for table default sort order

### LINKED ISSUE #6449: Default Table sort order
### What problem does this feature solve?
Enable developers to choose a default sorted column for a Table. At the moment, this can be forced manually by controlling the order of all columns using the `sortOrder` but this seems too much hassle for something that is desirable in a lot of tables.

### What does the proposed API look like?
Add 2 attributes to the Table-component. One which takes the `dataIndex` of the column to be sorted and the other the `defaultSortOrder` (ascend or descend).

### PR DESCRIPTION
**Pull request by megawac**
Some things to consider:

- what should be used to identify a column, in this PR we're using column title
- are the properties of the default sort order be documented correctly?
- should I of bothered to create and export this interface?

Fixes #6449

### CODE REVIEW

**Comment by yesmeck** [comment]
Hello @megawac , could you pr to branch feature-2.13?

File: components/table/index.en-US.md [hunk 1]
```diff
@@ -60,7 +60,8 @@ const columns = [{
 | rowKey        | get row's key, could be a string or function | string\|Function(record):string | 'key' |
 | rowClassName  | get row's className | Function(record, index):string | - |
 | expandedRowRender  | expanded container render for each row | Function | - |
-| defaultExpandedRowKeys | initial expanded row keys | string[] | - |
+| defaultSortOrder | default column sorting | Object({columnTitle, sortOrder}) | - |
```
Comment (benjycui) [inline components/table/index.en-US.md]: It should be `Column[defaultSortOrder]`, not `Table[defaultSortOrder]`.

**Comment by megawac** [comment]
Done @yesmeck

**Comment by benjycui** [comment]
@megawac you can rebase to fix CI.

**Comment by megawac** [comment]
Rebased on `feature-2.13`

**Comment by megawac** [comment]
That commit should fix it, just need to cleanup how the `ant-table-column-sort` was getting applied

**Comment by yesmeck** [comment]
I don't think `columnTitle` is good idea, if title is ReactNode this [line](https://github.com/megawac/ant-design/blob/3ba35d2ee20f74af201cc6bd8748c34152a5f3f3/components/table/Table.tsx#L347) will failed.

```javascript
const columns = [
  {
    title: <span>Age</span>,
    dataIndex: 'age',
    sorter: (a, b) => a.age - b.age,
  }
];
```

**Comment by afc163** [comment]
> It should be Column[defaultSortOrder], not Table[defaultSortOrder].

I think so.

**Comment by afc163** [comment]
We should revert it and create a new PR for `Column[defaultSortOrder]`.

**Comment by simondiep** [comment]
Instead of reverting the change, could the column `dataIndex` be specified, rather than the column title?  I like this at the `Table` level because you can pass it through as a prop, rather than having to make the set of static columns dynamic.

**Comment by viztor** [comment]
@afc163 It appears that the Table[defaultSortOrder] was removed and yet Column[defaultSortOrder] have not been introduced.

**Pull request merged**